man/common/alg.md

```diff
@@ -65,9 +65,10 @@ general format for specifying this data:
    This portion of the complex algorithm specifier is required. The remaining scheme and symmetric details
    will default based on the type specified and the type of the object being created.
 
-  * aes - Default AES: aes128cfb
+  * aes - Default AES: aes128
   * aes128`<mode>` - 128 bit AES with optional mode (*ctr*|*ofb*|*cbc*|*cfb*|*ecb*). If mode is not
-      specified, defaults to *cfb*.
+      specified, defaults to *null*.
```

Comment:
Looks like the code was setting the default to null and you're just correcting that issue in the man page, is that right?

Separate patch.

Reply:
Right.
I will provide a separate commit